app/js/components/partials/TagModal.react.js

```diff
@@ -7,6 +7,13 @@ import AuthActions from '../../actions/AuthActions'
 import webAPI from '../../utils/webAPI'
 import _ from 'jquery'
 
+/*
```

Comment:
💚 We should keep this up. Leaving comments in our functions makes it easier for other developers to get our logic